phpstan failed nightly

Build: https://drone.owncloud.com/owncloud/openidconnect/1738/2/5

```bash
+ make test-php-phpstan
/usr/bin/composer install
Installing dependencies from lock file (including require-dev)
Verifying lock file contents can be installed on current platform.
Package operations: 1 install, 0 updates, 0 removals
  - Downloading bamarni/composer-bin-plugin (1.8.2)
  - Installing bamarni/composer-bin-plugin (1.8.2): Extracting archive
Generating optimized autoload files
[bamarni-bin] The setting "extra.bamarni-bin.forward-command" will be set to "true" from 2.x onwards. If you wish to keep it to "false", you need to set it explicitly.
1 package you are using is looking for funding.
Use the `composer fund` command to find out more!
phpstan composer.lock is not up to date.
/usr/bin/composer bin phpstan install --no-progress
[bamarni-bin] Checking namespace vendor-bin/phpstan
No composer.lock file present. Updating dependencies to latest instead of installing from lock file. See https://getcomposer.org/install for more information.
Loading composer repositories with package information
Info from https://repo.packagist.org: #StandWithUkraine
Updating dependencies
Lock file operations: 1 install, 0 updates, 0 removals
  - Locking phpstan/phpstan (1.9.7)
Writing lock file
Installing dependencies from lock file (including require-dev)
Package operations: 1 install, 0 updates, 0 removals
  - Downloading phpstan/phpstan (1.9.7)
  - Installing phpstan/phpstan (1.9.7): Extracting archive
Generating autoload files
1 package you are using is looking for funding.
Use the `composer fund` command to find out more!

php -d zend.enable_gc=0 vendor-bin/phpstan/vendor/bin/phpstan analyse --memory-limit=4G --configuration=./phpstan.neon --no-progress --level=5 appinfo lib
 ------ --------------------------------------------- 
  Line   lib/Controller/LoginFlowController.php       
 ------ --------------------------------------------- 
  161    No error to ignore is reported on line 161.  
 ------ --------------------------------------------- 
 [ERROR] Found 1 error                                                          
make: *** [Makefile:128: test-php-phpstan] Error 1
```

--- a/lib/Controller/LoginFlowController.php
+++ b/lib/Controller/LoginFlowController.php
@@ -157,7 +157,6 @@ class LoginFlowController extends Controller {
 
 		// trigger login process
 		if ($this->userSession->createSessionToken($this->request, $user->getUID(), $user->getUID()) &&
-			// @phpstan-ignore-next-line because loginUser does not have the third argument in older versions
 			$this->userSession->loginUser($user, null, OpenIdConnectAuthModule::class)) {
 			$this->session->set('oca.openid-connect.id-token', $openid->getIdToken());
 			$this->session->set('oca.openid-connect.access-token', $openid->getAccessToken());

